apps/desktop/desktop_native/autotype/src/windows.rs

```diff
@@ -25,25 +25,29 @@ pub fn get_foreground_window_title() -> std::result::Result<String, ()> {
 
 /// Attempts to type the input text wherever the user's cursor is.
 ///
-/// `input` must be an array of utf-16 encoded characters to insert.
+/// `input` must be a vector of utf-16 encoded characters to insert.
+/// `keyboard_shortcut` must be a vector of Strings, where valid shortcut keys: Control, Alt, Super, Shift, letters a - Z
 ///
 /// https://learn.microsoft.com/en-in/windows/win32/api/winuser/nf-winuser-sendinput
-pub fn type_input(input: Vec<u16>) -> Result<(), ()> {
-    const TAB_KEY: u16 = 9;
+pub fn type_input(input: Vec<u16>, keyboard_shortcut: Vec<String>) -> Result<(), ()> {
```

Comment:
nit: instead of moving the vecs here, we can pass by reference because the usage of them does not require ownership or to consume the vec. The pass by reference is a bit more optimized than the move and is more idiomatic / future proof since ownership isn't required here.


```suggestion
pub fn type_input(input: &Vec<u16>, keyboard_shortcut: &Vec<String>) -> Result<(), ()> {
```